src/utils/context/AuthContext.jsx

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable react/prop-types */
+import { createContext, useState, useEffect } from 'react'
+import { jwtDecode } from "jwt-decode";
+import { Outlet, useNavigate } from 'react-router-dom';
+
+const AuthContext = createContext()
+
+export default AuthContext;
+
+
+
+export const AuthProvider = () => {
+    let [authTokens, setAuthTokens] = useState(()=> localStorage.getItem('authTokens') ? JSON.parse(localStorage.getItem('authTokens')) : null)
+    let [user, setUser] = useState(()=> localStorage.getItem('authTokens') ? jwtDecode(localStorage.getItem('authTokens')) : null)
+    let [loading, setLoading] = useState(true)
+
+    const navigate = useNavigate()
+
+    let loginUser = async (e )=> {
+        e.preventDefault()
+        let response = await fetch('http://127.0.0.1:8000/token/', {
```

Comment:
Don't use the url like this! 
Merge develop to your branch to get the docker integration and use /api/token/ instead.
```suggestion
        let response = await fetch('/api/token/', {
```